test/python/api_config.py

```diff
@@ -86,6 +86,30 @@ def get_api_client(username='admin', role='user'):
     client.configuration.api_key = {'Authorization': f'Token token="{api_token}"'}
     return client
 
+OIDC_POLICY_FILE = 'test/config/oidc-webservice.yml'
```

Comment:
I would toss this either at the top of the file with the rest of the paths/constants or in the `setup_oidc_webservice` function